### PR TITLE
Highlight filter icon when it's active

### LINKED ISSUE #1181: Log search filter background color
We should try to give a background color to the filter icon when selected.

STR:

1. Navigate to https://taskcluster-ui.herokuapp.com/tasks/Kp5viqUERuKejEkgz8u4dA/runs/0/logs/live/https%3A%2F%2Fqueue.taskcluster.net%2Fv1%2Ftask%2FKp5viqUERuKejEkgz8u4dA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log

Screen Shot 2019-08-14 at 2 19 57 PM

2. Click on the filter button (4 lines icon)

Actual:

Screen Shot 2019-08-14 at 2 20 03 PM

Expected: The filter should have a background similar to the first button in the following screenshot:

Screen Shot 2019-08-14 at 2 22 03 PM

### PR DESCRIPTION
**Pull request by arku**
Fixes #1181 

Screenshots

image

image

### CODE REVIEW

File: ui/src/components/Log/index.jsx [hunk 1]
```diff
@@ -19,89 +19,105 @@ const LINE_NUMBER_MATCH = /L(\d+)-?(\d+)?/;
...
-      paddingBottom: theme.spacing.unit,
-      color: theme.palette.text.primary,
-      '-webkit-font-smoothing': 'auto',
+@withStyles(theme => {
+  const filterStyles = {
+    padding: '4px 8px',
+    lineHeight: 1,
+    '&.active': {
+      ...theme.mixins.successIcon,
+      borderRadius: '4px',
```
Comment (helfi92) [inline ui/src/components/Log/index.jsx]: The border radius could be set for all times, not just when the button is active. Also, we can omit the px and remove the quotations: `borderRadius: 4,`.

File: ui/src/components/Log/index.jsx [hunk 2]
```diff
@@ -19,89 +19,105 @@ const LINE_NUMBER_MATCH = /L(\d+)-?(\d+)?/;
...
+    fabIcon: {
+      ...theme.mixins.fabIcon,
+    },
+    logSpeedDial: {
+      ...theme.mixins.fab,
+      ...theme.mixins.actionButton,
+      bottom: theme.spacing.triple,
+    },
+  };
+})
```
Comment (helfi92) [inline ui/src/components/Log/index.jsx]: Any chance you could give an overview of the changes you made? The diff is a bit hard to follow for some reason 😅

**Comment by arku** [comment]
> Let's make the fill white instead of black when the button is active:
> 
> Screen Shot 2019-08-15 at 10 06 05 AM
> 
> If we could also do the same for the follow log button, that'd be great :)

I left it black to be consistent with the follow log button styles. Let me change them both to white.

File: ui/src/components/Log/index.jsx (hunk 1 above)
Comment (arku) [reply ui/src/components/Log/index.jsx]: Done.

File: ui/src/components/Log/index.jsx (hunk 2 above)
Comment (arku) [reply ui/src/components/Log/index.jsx]: Sure. I added a `filterStyles` object [here](https://github.com/taskcluster/taskcluster/pull/1183/files#diff-6af56ca36cff0662454eca418e5191b6R24) for the active filter styles in both dark and light themes. And I updated the follow log button styles [here](https://github.com/taskcluster/taskcluster/pull/1183/files#diff-6af56ca36cff0662454eca418e5191b6R110). 

Does that help?

**Comment by arku** [comment]
@helfi92 I have addressed your comments. Do you mind taking another look?

**Comment by arku** [comment]
@helfi92 It looks much better now 😍 

Thanks!
